nri-kafka/src/Kafka/Worker/Internal.hs

```diff
@@ -203,7 +316,7 @@ rebalanceCallback skipOrNot observability callback offsetSource state consumer r
             log <- Platform.silentHandler
             fetchResult <- Task.attempt log (fetch newPartitions)
             case fetchResult of
-              Err err -> Exception.throwIO err
+              Err err -> Exception.throwString (Text.toList err)
```

Comment:
why string?

Reply:
I changed `err` to be of type `Text`, which doesn't implement `Exception`, so we weren't able to use `throwIO` on it anymore.